halium-install

```diff
@@ -74,9 +74,11 @@ spinner $!
 echo "I: Shrinking images"
 shrink_images
 
-echo "I: Pushing rootfs and android image to /data via ADB"
-if ! flash $ROOTFS_RELEASE; then
-	echo "Error: Couldn't copy the files to the device, is it connected?"
+echo "I: Pushing rootfs and android image to /data"
+if [ $FLASH_ADB ]; then
+	flash_error flash_adb
+else
+	flash_error flash_rsync
```

Comment:
Where are the parameters 'flash_adb' and 'flash_rsync' used?

Reply:
The flash_error functions calls the flash_rsync function